cnv_germline_case_scattered_workflow.inputs.json

```diff
@@ -0,0 +1,37 @@
+{
+  "CNVGermlineCaseScatteredWorkflow.normal_bams": [
+    "gs://gatk-best-practices/cnv_germline_pipeline/SM-74NEG_20xy-downsampled.bam",
```

Comment:
I have the same concerns about these files that I raised in https://github.com/gatk-workflows/gatk4-somatic-cnvs/pull/1#discussion_r165184917 and on Slack.  Let's just use 2-3 public germline WES plumbing BAMs.  What BAMs are used to test other WES pipelines?  Can we try to congregate on a common set of testing BAMs?

Reply:
Also, where do we want to document parameter suggestions for WES/WGS?  Note previous discussion at https://github.com/broadinstitute/gatk/issues/4719.